cli/src/main/java/com/box/l10n/mojito/cli/command/checks/EmptyPlaceholderChecker.java

```diff
@@ -0,0 +1,76 @@
+package com.box.l10n.mojito.cli.command.checks;
+
+import com.box.l10n.mojito.cli.command.extraction.AssetExtractionDiff;
+import com.box.l10n.mojito.regex.PlaceholderRegularExpressions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * {@link AbstractCliChecker} that verifies a source string does not contain an empty placeholder e.g. '{}'.
+ *
+ * The check uses Single or Double braces regex to identify placeholders in a source string and then checks that the
+ * placeholder substring contains regex 'word' values.
```

Comment:
I'm not sure that all non whitespace characters are valid placeholder values but it's beyond the scope of this check to verify placeholder validity. If an invalid placeholder is provided I would expect this will cause runtime errors at the source code level anyway which a dev would need to fix.